pkg/controller/workload/job/job_controller.go

```diff
@@ -151,45 +148,41 @@ func (h *parentWorkloadHandler) queueReconcileForChildJob(object client.Object,
 	}
 }
 
-type BatchJob struct {
+type Job struct {
```

Comment:
Should we change this to a general name `Job`? Considering we may want to change the folder `workload` to `job` in the future.

Reply:
I'm ok with either Job or BatchJob. It was @alculquicondor suggestion previously to rename just to Job. This makes sense, from the consistency perspective (we also have MPIJob, not KubeflowMPIJob). 
As for the folder names, maybe we could have something like:
```
controller/
├── jobframework
├── jobs/
   ├── mpijob
   ├── job
```

Reply:
It should be fine either way.